homework-submission/class16-week1/src/components/Todo.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+class Todo extends React.Component {
+  render() {
+    const { description, deadline } = this.props;
+    return (
+      <li className="todo_item">
+        <h2>
```

Comment:
We usually use `<h2>` and `<h3>` if we have **Headers** but in this case this is normal text. So keep it simple

```
<li className="todo_item">
  <span className="desc_label">Description</span>
  <span className="desc_value">{description}</span>
</li>
```

In css you can add whatever style you like using the classes.

Reply:
this has been amended as well.